ipa/ipa_test.go

```diff
@@ -12,10 +12,8 @@ import (
 	"github.com/crate-crypto/go-ipa/test_helper"
 )
 
-var ipaConf = NewIPASettings()
```

Comment:
In this file, I just removed this global. While I understand the idea is to only calculate this once for the two tests, it isn't very pleasant since if I want to run other tests in the same package that don't use it, we still have to wait while this is generated.

This was now moved just to the tests that required them.

Reply:
I think it was taking a while to compute it -- if it still takes too long you can possibly do t.Parallel()

Reply:
Yes, actually tried, but doing that requires a lot of memory from the host, since these tables are pretty big. :)
What we can do is `sync.Once` or similar style... so gets executed when running the test that requires them, but avoid doubling work.